corehq/apps/locations/templates/locations/manage/partials/drilldown_location_widget.html

```diff
@@ -34,6 +34,14 @@
             $('#{{ css_id }}').val(null);
         }
     }).trigger('change');
+
+    $('#{{ submit_form_id }}').on('submit', function() {
+         // Notify user in case the source list has options but none selected
+         if(!!$('#{{ source_css_id }}').val() && !$('#{{ css_id }}').val()) {
+             alert("{{ empty_primary_location_message|safe }}");
+             return false;
+         }
+    })
```

Comment:
I think we discourage using `alert`, is there another way to cue the user?  This could mark the widget with an `.error` class so it's red and show the help text underneath.  Alternatively, it could `.disable` the submit button and warn of the error.